components/inferno/src/Particles.tsx

```diff
@@ -9,6 +9,9 @@ interface MutableRefObject<T> {
 	current: T | null;
 }
 
+/**
+ * @param {{id?: string,width?: string,height?: string,options?: ISourceOptions,params?: ISourceOptions,style?: CSSProperties,className?: string,canvasClassName?: string,container?: RefObject<Container>}}
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/matteobruni/tsparticles/pullRequest?prid=6397531)